### ISSUE TITLE
量化过程中调用quantizer = torch_quantizer(quant_mode, model, (input))函数的问题

### ISSUE THREAD

**Issue by xiehou123456** [xiehou123456/ganzhi#6]
在量化过程中，调用quantizer = torch_quantizer(quant_mode, model, (input))函数的时候。遇到RuntimeError: Only tuples, lists and Variables supported as JIT inputs/outputs. Dictionaries and strings are also accepted but their usage is not recommended. But got unsupported type numpy.ndarray问题
![image](https://user-images.githubusercontent.com/20207901/120060412-56403380-c08a-11eb-8619-9d1de1e269f3.png)

**Comment by xiehou123456** [comment]
base_model.py 198行return out, cur_feat[0].detach().cpu().numpy()改成return out,cur_feat[0].detach()
